src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -293,7 +254,7 @@ export class Accept extends Component<AcceptProps, AcceptState> {
 }
 
 export const AcceptFragmentContainer = createFragmentContainer(
-  injectDialog(trackPageViewWrapper(Accept)),
+  injectCommitMutation(injectDialog(trackPageViewWrapper(Accept))),
```

Comment:
Maybe 
```js
import { compose } from 'lodash'

export const AcceptFragmentContainer = createFragmentContainer( 
  compose(
     injectCommitMutation, 
     injectDialog, 
     trackPageViewWrapper
  )(trackPageViewWrapper)
)
```
or some variant thereof (since this is getting kinda windy, though not sure if this creates typing headaches).

Either way just a pattern thought, not a blocker.